src/index.mjs

```diff
@@ -75,6 +73,16 @@ export default function (options) {
     timeoutFn: requestIdleCallback,
     el: document,
   }, options);
+  
+  // actually initialize intersection observer here
+  observer = new IntersectionObserver(entries => {
+    entries.forEach(entry => {
+      if (entry.isIntersecting) {
+        const url = entry.target.href;
+        if (toPrefetch.has(url)) prefetcher(url);
+      }
+    });
+  }, observerConfig);
```

Comment:
Let's replace this with a `config` object that allows the user to directly manipulate the IO settings. 

```suggestion
  }, options.config || {});
```

The only drawback here is that it requires the user to know _what_ those options are... but we can easily link to [its options](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API#Intersection_observer_options) in the docs IMO. Plus, I personally prefer to expose native options, rather than reinvent them with new names.

What do you think @addyosmani ?

Reply:
Yeah, I was gonna do that. But I didn't wanna step add something to options without advisement. I'll make that change.

Reply:
I just wanted to add that I'm in support of exposing native names vs defining our own variants where possible 👍

Thanks for working on this, @aretheregods!